### PR TITLE
feat(Learn): update metadata of Learn section

### PR DESCRIPTION
**Pull request by ashucoder9**
This PR updates the metadata of the "Learn" section in documentation.

### CODE REVIEW

File: docs/learn/avalanche/avalanche-consensus.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ---
-description: Avalanche is a consensus protocol that is scalable, robust, and decentralized.
+tags: [consensus, snowball, protocol]
```
Comment (meaghanfitzgerald) [inline docs/learn/avalanche/avalanche-consensus.md:2]: Tags are visible links embed at the bottom of each page and should only include the domains the page lives under. Words like this should be in the `keywords` metadata field. See [here](https://github.com/ava-labs/avalanche-docs/blob/master/docs/build/subnet/deploy/custom-vm-subnet.md) for a good example.

File: docs/learn/avalanche/avalanche-platform.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ---
-description: Avalanche features 3 built-in blockchains which includes Exchange Chain (X-Chain), Platform Chain (P-Chain), and Contract Chain (C-Chain).  More info here.
+tags: [Primary Network, P-Chain, X-Chain, C-Chain]
```
Comment (meaghanfitzgerald) [inline docs/learn/avalanche/avalanche-platform.md:2]: These are great

File: docs/learn/avalanche/avalanche-consensus.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
...
+description: Avalanche Conesensus is a consensus protocol that is scalable, robust, and decentralized.
+keywords: [avalanche consensus, snowball, scaling blockchains, nakamoto consensus]
+sidebar_label: Avalanche Consensus
```
Comment (meaghanfitzgerald) [inline docs/learn/avalanche/avalanche-consensus.md:5]:
```suggestion
tags: [Avalanche Concensus, Avalanche Protocol]
description: Avalanche Consensus is a consensus protocol that is scalable, robust, and decentralized.
keywords: [avalanche consensus, snowman concensus, scaling blockchains, nakamoto consensus]
sidebar_label: Avalanche Consensus
```

File: docs/learn/avalanche/avax.md [hunk 4]
```diff
@@ -1,6 +1,10 @@
 ---
+tags: [Avax, Tokenomics]
+description: AVAX is the limited-supply token of the Avalanche network, used to cover fees, enhance security via staking, and facilitate transactions across its various Subnets.
+keywords: [docs, avalanche, avax token, tokenomics, avax utility]
```
Comment (meaghanfitzgerald) [inline docs/learn/avalanche/avax.md:4]:
```suggestion
tags: [Avax, Tokenomics]
description: AVAX is the limited-supply token of the Avalanche network, used to cover fees, enhance security via staking, and facilitate transactions across its various Subnets.
keywords: [documentation, avalanche, avax token, tokenomics, avax utility]
```

File: docs/learn/avalanche/awm.md [hunk 5]
```diff
@@ -1,7 +1,11 @@
 ---
+tags: [Avalanche Warp Messaging, cross-subnet communication]
+description: AWM allows seamless communication between different subnetworks on Avalanche, enabling developers to establish custom communication protocols.
+keywords: [docs, avalanche, avalanche warp messaging, awm, cross-subnet communication]
```
Comment (meaghanfitzgerald) [inline docs/learn/avalanche/awm.md:4]:
```suggestion
tags: [Avalanche Warp Messaging, Cross-Subnet Communication]
description: AWM allows seamless communication between different subnetworks on Avalanche, enabling developers to establish custom communication protocols.
keywords: [docs, documentation, avalanche, avalanche warp messaging, awm, cross-subnet communication, cross-chain, ]
```

File: docs/learn/avalanche/intro.md [hunk 6]
```diff
@@ -1,5 +1,7 @@
 ---
 slug: /intro
+description: Avalanche is an open-source platform for building dApps in an interoperable, decentralized, and highly scalable ecosystem.
+keywords: [docs, avalanche, scalability, build on avalanche, dApps]
 sidebar_label: 🔺 What is Avalanche?
```
Comment (meaghanfitzgerald) [inline docs/learn/avalanche/intro.md:5]: Please add tags

File: docs/learn/avalanche/subnets-overview.md [hunk 7]
```diff
@@ -1,4 +1,5 @@
 ---
+keywords: [docs, avalanche, subnets, sovereign blockchains, interoperability]
 sidebar_label: Subnets
```
Comment (meaghanfitzgerald) [inline docs/learn/avalanche/subnets-overview.md:3]: please add tags and description

File: docs/learn/avalanche/virtual-machines.md [hunk 8]
```diff
@@ -1,3 +1,9 @@
+---
+tags: [Virtual Machines, Subnet-EVM]
+keywords: [docs, avalanche, virtual machines, subnets, Subnet EVM]
+sidebar_label: Virtual Machines
```
Comment (meaghanfitzgerald) [inline docs/learn/avalanche/virtual-machines.md:4]: please add a description

File: docs/learn/projects.md [hunk 9]
```diff
@@ -1,3 +1,9 @@
+---
+keywords: [docs, avalanche, AvalancheGO, avalanche CLI, Core Ethereum, Subnet EVM]
+sidebar_label: Projects
```
Comment (meaghanfitzgerald) [inline docs/learn/projects.md:3]: This page can be deleted, and redirected to `https://docs.avax.network/intro` via the `_redirects` file

**Comment by ashucoder9** [comment]
@meaghanfitzgerald thanks for the feedback. You may review now ✅

File: docs/learn/avalanche/avalanche-consensus.md (hunk 1 above)
Comment (ashucoder9) [reply docs/learn/avalanche/avalanche-consensus.md]: Gotcha ✅

File: docs/learn/projects.md (hunk 9 above)
Comment (meaghanfitzgerald) [reply docs/learn/projects.md:3]: doesn't seem like the change was made

Comment (meaghanfitzgerald) [reply docs/learn/projects.md:3]: committed the change directly to the branch

**Review by meaghanfitzgerald — DISMISSED** [review]

**Review by meaghanfitzgerald — APPROVED** [review]

**Review by meaghanfitzgerald — APPROVED** [review]

**Comment by meaghanfitzgerald** [comment]
I removed the projects page, added to the `_redirects` file, removed the path from `sidebars.json`, and fixed all broken links. Take a look if you have time at the additional changes, it is good to know for further contribution. lgtm

**Pull request merged**